modules/js/index.js

```diff
@@ -94,45 +100,36 @@ bookForm.addEventListener('submit', (e) => {
 
 window.addEventListener('DOMContentLoaded', book.showBook());
 
-const navlist = document.querySelector('#show-list-button');
-const navadd = document.querySelector('#add-new-button');
-const navcon = document.querySelector('#contact-button');
-
-const list = document.querySelector('#list');
-const addNew = document.querySelector('#add');
-const connav = document.querySelector('#contact');
-
-function listfun(e) {
+const listfun = (e) => {
   if (e.currentTarget === navlist) {
     list.style.display = 'flex';
     addNew.style.display = 'none';
     connav.style.display = 'none';
   }
-}
+};
 
-function addfun(e) {
+const addfun = (e) => {
   if (e.currentTarget === navadd) {
     list.style.display = 'none';
     addNew.style.display = 'flex';
     connav.style.display = 'none';
   }
-}
+};
 
-function confun(e) {
+const confun = (e) => {
   if (e.currentTarget === navcon) {
     list.style.display = 'none';
     addNew.style.display = 'none';
     connav.style.display = 'flex';
   }
-}
+};
 
 navlist.addEventListener('click', listfun);
 
 navadd.addEventListener('click', addfun);
 
 navcon.addEventListener('click', confun);
 
-// eslint-disable-next-line no-unused-vars
 const displayTime = () => {
   document.getElementById('current-date').innerText = new Date().toLocaleString();
   setTimeout(displayTime, 1000);
```

Comment:
- [ ] Great job with implementing the display date function 🙌. However, you should use the [Luxon](https://moment.github.io/luxon/#/install) library to display the time ⌚. Please kindly install this package and use it to display the time, for more information use this article: [Get the current date and time](https://moment.github.io/luxon/#/tour?id=get-the-current-date-and-time).

Reply:
Thanks for the information. I am going to use that now